src/core/syntax-tree/lexicon.ts

```diff
@@ -31,10 +31,21 @@ const isPrime = (c: number) => {
     return true;
 };
 
-const baseOperand = {
+const operand = {
     resolve: (target: any, tree: ISyntaxTree) => {
-        Logger.trace('resolve -> exp', target, tree);
+        if (/^('.*')|(".*")$/.test(tree.value)) {
+            return target[
+                tree.value.slice(1, tree.value.length - 1)
+            ];
+        } else if (/^(\w|[:.\-+])+$/.test(tree.value)) {
+            return target[tree.value];
+        }
+    },
+    regexp: /^('([^()']|\\')+'|"([^()"]|\\")+"|(\w|[:.\-+])+)/
+};
```

Comment:
Separate operand and expression logic -- support arbitrary quoted string or limited character set without quotes.

Reply:
`[^()']` - why are `()` excluded?

Reply:
No good reason. Removing the exclusion.